pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -256,10 +256,10 @@ func (cb *ClusterBuilder) buildDefaultCluster(name string, discoveryType cluster
 // requires a single protocol per port, and the DestinationRule issue is slated to move to Sidecar.
 // Note: clusterPort and instance.Endpoint.EndpointPort are identical for standard Services; however,
 // Sidecar.Ingress allows these to be different.
-func (cb *ClusterBuilder) buildInboundClusterForPortOrUDS(proxy *model.Proxy, clusterPort int, bind string,
+func (cb *ClusterBuilder) buildInboundClusterForPortOrUDS(clusterPort int, bind string,
```

Comment:
hope we donot need to add it back again, lol

Reply:
If we need it, we should use ClusterBuilder's proxy - instead of passing it as arg here :-)